Model/Reclaim.php

```diff
@@ -106,6 +111,52 @@ public function product($quote_id, $item_id) {
         return $response;
     }
 
+    /**
+    * @return mixed
+    */
+    public function backinstock($product_id)
+    {
+        if (!$product_id){
+            throw new NotFoundException(_('A product id is required'));
+        }
+        $product = $this->_objectManager->create('Magento\Catalog\Model\Product')->load($product_id);
+
+        if (!$product){
+            throw new NotFoundException(_('A product was not found'));
+        }
+
+        $productId = $product->getId();
+
+        $response = array(array(
+            'id' => $productId,
```

Comment:
![image](https://user-images.githubusercontent.com/25993138/58294415-f9715a80-7d97-11e9-8d6a-033980d1369c.png)


It looks like Magento's API doesn't support IDs and only supports SKUs. 
https://devdocs.magento.com/guides/v2.3/rest/list.html

Can you please change this to:
```suggestion
            'id' => $product->getSku(),
```

Reply:
Aargh... 🤕We need a product ID, which we can use to find its SKU. Let's keep this as is. But we still need to pass variant SKU.

Reply:
![image](https://user-images.githubusercontent.com/25993138/58295541-bdd98f00-7d9d-11e9-80aa-3ba171188728.png)

I think we also need to pass the store ID for both product and variant... 🤢

Reply:
good call on store id, will make it optional arg